### PR TITLE
pep8 and docstring untruths

### PR DESCRIPTION
**Pull request by marqh**
non-functional change to pp.py

pep8 plus removal of claims about save and as_pairs, as_fields being able to take CubeLists as inputs

### CODE REVIEW

File: lib/iris/fileformats/pp.py [hunk 1]
```diff
@@ -2223,7 +2317,7 @@ def as_fields(cube, field_coords=None, target=None):
     Args:
 
     * cube:
-        A :class:`iris.cube.Cube`, :class:`iris.cube.CubeList` or list of cubes.
+        A :class:`iris.cube.Cube`
```
Comment (marqh) [inline lib/iris/fileformats/pp.py]: this is a material docstring change

File: lib/iris/fileformats/pp.py [hunk 2]
```diff
@@ -2104,7 +2192,7 @@ def as_pairs(cube, field_coords=None, target=None):
...
```
Comment (marqh) [inline lib/iris/fileformats/pp.py]: this is a material docstring change

File: lib/iris/fileformats/pp.py [hunk 3]
```diff
@@ -2075,19 +2159,23 @@ def save(cube, target, append=False, field_coords=None):
 
     Args:
 
-        * cube         - A :class:`iris.cube.Cube`, :class:`iris.cube.CubeList` or list of cubes.
+        * cube         - A :class:`iris.cube.Cube`
```
Comment (marqh) [inline lib/iris/fileformats/pp.py]: this is a material docstring change

**Comment by pp-mo** [comment]
Thanks @marqh.
A good spot, and a long-overdue tidy !

**Pull request merged**